nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/test/java/org/apache/nifi/distributed/cache/server/map/TestDistributedMapServerAndClient.java

```diff
@@ -312,6 +355,27 @@ private DistributedMapCacheClientService createClient(final int port) throws Ini
         return client;
     }
 
+    private Thread startServerSocket(ServerSocketChannel serverSocketChannel) {
+        final Runnable runnable = new Runnable() {
+            @Override
+            public void run() {
+                while (true) {
+                    final SocketChannel socketChannel;
```

Comment:
I don't think we need this line.
line in try could be:
serverSocketChannel.accept();